registry/api/errcode/errors.go

```diff
@@ -97,25 +101,28 @@ type Error struct {
 
 var _ error = Error{}
 
-// ErrorCode returns the ID/Value of this Error
+// ErrorCode returns the ID/Value of this Error.
 func (e Error) ErrorCode() ErrorCode {
 	return e.Code
 }
 
 // Error returns a human readable representation of the error.
 func (e Error) Error() string {
-	return fmt.Sprintf("%s: %s",
```

Comment:
Please leave this: it tells use what the code and message are in a nice human readable format.